fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/Lambda.java

```diff
@@ -41,21 +41,22 @@ public class Lambda extends Expression {
 
     private final List<String> argumentNames;
 
+    private final Expression lambdaFunctionBody;
+
     /**
      * constructor
      */
     public Lambda(List<String> argumentNames, Expression lambdaFunction) {
-        this(argumentNames, ImmutableList.of(lambdaFunction));
-    }
-
-    public Lambda(List<String> argumentNames, Expression lambdaFunction, List<ArrayItemReference> arguments) {
-        this(argumentNames, ImmutableList.<Expression>builder().add(lambdaFunction).addAll(arguments).build());
+        this.argumentNames = ImmutableList.copyOf(Objects.requireNonNull(
+                argumentNames, "argumentNames should not null"));
```

Comment:
Call a unified constructor in other constructors instead of initializing member variables in each constructor.